tests/conftest.py

```diff
@@ -363,7 +363,10 @@ def cpu_instance_type(sagemaker_session, request):
 
 @pytest.fixture(scope="module")
 def gpu_instance_type(request):
-    return "ml.p2.xlarge"
+    return "ml.p3.2xlarge"
+
+
+#     return "ml.p2.xlarge"
```

Comment:
Please remove uncommented code if not required, as well as extra newlines at end.

Reply:
addressed in the current revision.